.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,19 @@
+# What steps does it take to reproduce the issue?
```

Comment:
Should this be the h1? The feature request template has "overview" as the first line.

Reply:
I have decided to use `##` (h2) in all section titles of the templates. So the titles are not too big. This is how I've often seen it in Dataverse issues.